### PR TITLE
feat(project-access): support freestyle CAP

### PR DESCRIPTION
**Pull request by PetrStribrny**
Add freestyle CAP app support.

### CODE REVIEW

**Review by nahmed22 — DISMISSED** [review]

**Review by Klaus-Keller — CHANGES_REQUESTED** [review]
@PetrStribrny, the change set is missing. This should be a minor change.

**Comment by PetrStribrny** [comment]
> @PetrStribrny, the change set is missing. This should be a minor change.

@Klaus-Keller changeset added, please check if `patch` ic correct type. Thanks.

File: .changeset/short-planes-fix.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': patch
```
Comment (Klaus-Keller) [inline .changeset/short-planes-fix.md:2]: @PetrStribrny as mentioned, I think this is minor as the result of call `getProject()` as more data. 
```suggestion
'@sap-ux/project-access': minor
```

Comment (PetrStribrny) [reply .changeset/short-planes-fix.md:2]: > @PetrStribrny as mentioned, I think this is minor as the result of call `getProject()` as more data.

@Klaus-Keller done

**Review by Klaus-Keller — APPROVED** [review]
Thanks @PetrStribrny,
- code changes to include freestyle applications look good
- changeset is correct
- test coverage is good
- did local test, no issues found

Approved from my side.

**Review by nahmed22 — APPROVED** [review]
Looks good to me, 

Good test coverage and changeset maintained.

**Pull request merged**